APMrover2/mode_guided.cpp

```diff
@@ -31,8 +31,8 @@ void ModeGuided::update()
                 calc_steering_to_waypoint(_reached_destination ? rover.current_loc : _origin, _destination, _reversed);
                 calc_throttle(calc_reduced_speed_for_turn_or_distance(_reversed ? -_desired_speed : _desired_speed), true, true);
             } else {
-                if (rover.is_boat()) {
-                    start_loiter();
+                if (rover.is_boat() && start_loiter()) {
+                    gcs().send_text(MAV_SEVERITY_INFO, "Loiter in GUIDED mode");
```

Comment:
Hi there.  I'd prefer if we don't send this text to the GCS.  Recently I'm pushing to only send messages when unusual things are happening or errors.  So when all is working OK, let's not send messages.

Reply:
@rmackay9 Ok. I'll fix it. My idea was to let the user know that it was Loiter that it was stop_vehicle so far.